src/messages/validation-failed-error-messages.js

```diff
@@ -0,0 +1,197 @@
+    message: 'This value is not a valid routing number'
+  },
+  GreaterThan: {
+    args: ['threshold'],
+    code: 'greater_than',
+    message: 'This value should be greater than {threshold}'
+  },
+  GreaterThanOrEqual: {
+    args: ['threshold'],
+    code: 'greater_than_or_equal_to',
+    message: 'This value should be greater than or equal to {threshold}'
+  },
+  Hash: {
+    code: 'invalid_hash',
+    message: 'This value is not a valid hash'
+  },
+  HaveProperty: {
+    code: 'required',
+    message: 'This value is required'
+  },
+  InvalidTaxId: {
+    code: 'invalid_tax_id',
+    message: 'The tax id is invalid'
+  },
+  Length: {
+    args: ['min', 'max'],
+    code: 'length',
+    message: 'This value must have between {min} and {max} characters'
+  },
+  LessThan: {
+    args: ['threshold'],
+    code: 'less_than',
+    message: 'This value should be less than {threshold}'
+  },
+  NotAlreadyInUse: {
+    code: 'unique',
+    message: 'This value should not be already in use'
+  },
+  NotBlank: {
+    code: 'not_blank',
+    message: 'This value should not be blank'
+  },
+  NotEmpty: {
+    code: 'not_empty',
+    message: 'This value should not be empty'
+  },
+
+  NotNull: {
+    code: 'not_null',
+    message: 'This value should not be null'
+  },
+  NotPoBox: {
+    code: 'not_po_box',
+    message: 'This value should be a PO Box'
+  },
+  NullOrString: {
+    args: ['min', 'max'],
+    code: 'null_or_string',
+    message: 'This value must be null or a string'
+  },
+  PasswordResetRestriction: {
+    args: ['recentPasswordRestrictionEndDate', 'threshold', 'unit'],
+    code: 'password_reset_restriction',
+    message: 'The user password has been changed in the last {threshold} {unit}'
+  },
+  PasswordUnique: {
+    args: ['createdAt'],
+    code: 'password_unique',
+    message: 'This password has already been used'
+  },
+  PciDss: {
+    code: 'pci_dss',
+    message: 'This value is not pci-dss compliant'
+  },
+  PersonName: {
+    code: 'name',
+    message: 'This value contains symbols or numbers that are not valid'
+  },
+  SubscriptionInactive: {
+    code: 'subscription_inactive',
+    message: 'The subscription should be inactive'
+  },
+  SufficientFunds: {
+    code: 'sufficient_funds',
+    message: 'Not enough funds for the specified amount'
+  },
+  SufficientUnits: {
+    code: 'sufficient_units',
+    message: 'There are no sufficient units to perform the required transaction'
+  },
+  SufficientUnlockedFunds: {
+    args: ['availableAt', 'currency', 'missing'],
+    code: 'sufficient_unlocked_funds',
+    message: 'You will have sufficient funds by {availableAt}'
+  },
+  Unique: {
+    args: ['key'],
+    code: 'unique',
+    message: 'This value must only contain unique values'
+  },
+  UnofficialFundSubscription: {
+    code: 'unofficial_fund_subscription',
+    message: 'This fund subscription should not be official'
+  },
+  UserMissingDocumentation: {
+    code: 'kyc_process_closed',
```

Comment:
Some of these messages look like they belong to specific projects and not to a generic library. Remove them, and let each project add their own messages?